### ISSUE TITLE
Frontend Slowness/Chopiness

### ISSUE THREAD

**Issue by Anto79-ops** [home-assistant/frontend#17796]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Hard to get logs for this, but since updating to beta, I've noticed that the frontend becomes slow and choppy after perusing through the pages. It then recovers, and then becomes slow and choppy again.

Only logs I have in my logs related to frontend are the "Unhandled promise rejection from Chrome WebView"

I'd be happy to share those logs if some are related to core, such as:

```
2023-09-02 17:42:19.759 ERROR (MainThread) [frontend.js.latest.202309010] Unhandled promise rejection from Chrome 116.0.0.0 on Windows 10
TypeError: Cannot destructure property 'scrollTop' of 'this._scrollerController' as it is null.
scrollTop (src/Virtualizer.ts:741:13)
_correctScrollError (src/Virtualizer.ts:546:9)
_finishDOMUpdate (src/Virtualizer.ts:538:9)
_updateDOM (src/Virtualizer.ts:603:11)
_handleLayoutMessage (src/Virtualizer.ts:441:43)
_hostSink (src/layouts/shared/BaseLayout.ts:478:9)
_sendStateChangedMessage (src/layouts/shared/BaseLayout.ts:364:9)
_reflow (src/layouts/flow.ts:491:10)
_reflow (src/layouts/shared/BaseLayout.ts:257:11)
reflowIfNeeded (src/Virtualizer.ts:563:20)
```

### Describe the behavior you expected

no choppiness, and smooth surfing like in 2023.8.4

### Steps to reproduce the issue

1. click on pages in HA, like settings, integrations, ZHA, etc. after 30 seconds to 1 minute
2.
3.
...

### What version of Home Assistant Core has the issue?

2023.9.0b

### What was the last working version of Home Assistant Core?

2023.8.4

### In which browser are you experiencing the issue with?

Google Chrome and Android App

**Comment by mhoogenbosch** [comment]
Got the same issue. Really choppy for some reason. Can't seem to find out what's going on. 

Already did a ha core rebuild, but did not help.

**Comment by kukulich** [comment]
Same problem in Firefox:

```
2023-09-03 10:37:53.696 ERROR (MainThread) [frontend.js.latest.202309010] Unhandled promise rejection from Firefox 117.0 on Windows 10
TypeError: this._scrollerController is null
this (src/Virtualizer.ts:741:38)
_correctScrollError (src/Virtualizer.ts:546:9)
_finishDOMUpdate (src/Virtualizer.ts:538:9)
_updateDOM (src/Virtualizer.ts:603:11)
_handleLayoutMessage (src/Virtualizer.ts:441:43)
_hostSink (src/layouts/shared/BaseLayout.ts:478:9)
_sendStateChangedMessage (src/layouts/shared/BaseLayout.ts:364:9)
_reflow (src/layouts/flow.ts:491:10)
_reflow (src/layouts/shared/BaseLayout.ts:257:11)
reflowIfNeeded (src/Virtualizer.ts:563:20)
2023-09-03 10:39:06.459 ERROR (MainThread) [frontend.js.latest.202309010] Unhandled promise rejection from Firefox 117.0 on Windows 10
TypeError: this._scrollerController is null
this (src/Virtualizer.ts:741:38)
_correctScrollError (src/Virtualizer.ts:546:9)
_finishDOMUpdate (src/Virtualizer.ts:538:9)
_updateDOM (src/Virtualizer.ts:603:11)
_handleLayoutMessage (src/Virtualizer.ts:441:43)
_hostSink (src/layouts/shared/BaseLayout.ts:478:9)
_sendStateChangedMessage (src/layouts/shared/BaseLayout.ts:364:9)
_reflow (src/layouts/flow.ts:491:10)
_reflow (src/layouts/shared/BaseLayout.ts:257:11)
reflowIfNeeded (src/Virtualizer.ts:563:20)
```

**Comment by mhoogenbosch** [comment]
Getting lots and lots of similar errors:

```
Logger: frontend.js.latest.202309010
Source: components/system_log/__init__.py:269
First occurred: 15:09:09 (28647 occurrences)
Last logged: 16:44:40

Unhandled promise rejection from Edge 116.0.1938.69 on Windows 10 "https://url/webrtc/webrtc-camera.js?v3.0.2"
Unhandled promise rejection from Edge 116.0.1938.69 on Windows 10 "https://url/hacsfiles/alarmo-card/alarmo-card.js?hacstag=391372854141"
Unhandled promise rejection from Edge 116.0.1938.69 on Windows 10 TypeError: Cannot read properties of undefined (reading 'min_temperature') FlowerCard.render (/local/lovelace-flower-card/flower-card.js:160:63) render (src/lit-element.ts:160:23) update (src/reactive-element.ts:1329:13) performUpdate (src/reactive-element.ts:1261:16) scheduleUpdate (src/reactive-element.ts:1233:24)
Unhandled promise rejection from Edge 116.0.1938.69 on Windows 10 Error: Cannot parse given Error object Object.parse (node_modules/error-stack-parser/error-stack-parser.js:35:22) parse (node_modules/stacktrace-js/stacktrace.js:106:61) fromError (node_modules/stacktrace-js/stacktrace.js:105:19) lines.push (src/resources/log-message.ts:37:40) ev.message.includes (src/state/logging-mixin.ts:44:25)
Unhandled promise rejection from Chrome WebView 116.0.0.0 on Android 12 TypeError: Cannot read properties of undefined (reading 'min_temperature') FlowerCard.render (/local/lovelace-flower-card/flower-card.js:160:63) render (src/lit-element.ts:160:23) update (src/reactive-element.ts:1329:13) performUpdate (src/reactive-element.ts:1261:16) scheduleUpdate (src/reactive-element.ts:1233:24)
```

And I know these are errors from custom cards, but never seen these before.

**Comment by Anto79-ops** [comment]
from what I understood, these new logs we are seeing were always there, just muted. The problem we are experiencing may not be related to these logs.....hence it makes this issue hard to track down.

All I know is, this was not a problem in 2023.8.4, so it must be related to 2023.9.0b

**Comment by mhoogenbosch** [comment]
Right, then it makes total sense i've not seen these messages before. But could it be the number of these messages is so extreme with a big dashboard, see above code, there have been ~29k occurrences, it just crashes or stops responding. 

My assumption is because of the fact i'm running into errors on my dashboard which have never been there either, see attached screens:

![image](https://github.com/home-assistant/frontend/assets/36899598/e0db2c0e-77b1-4da8-8dd3-e9ace2f5f4ef)

and from a broader perspective:

![image](https://github.com/home-assistant/frontend/assets/36899598/df5c46f7-e247-4850-88b4-3937f28576c0)

**Comment by silamon** [comment]
It becomes reproducible given that information. It keeps trying to send error messages to the system log even if Home Assistant is (temporarily) disconnected. It fails and raises yet another error message. It goes on until a reconnection can be made.

**Comment by bramkragten** [comment]
Fixed in #17818

**Comment by Anto79-ops** [comment]
Thank you very much!
